pkg/apis/ssp/v1/ssp_types.go

```diff
@@ -4,14 +4,18 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+type TemplateValidator struct {
+	// Replicas is the number of replicas of the template validator pod
+	Replicas int32 `json:"replicas"`
+}
 
 // SSPSpec defines the desired state of SSP
 type SSPSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
+
+	// TemplateValidator is configuration of the template validator operand
+	TemplateValidator TemplateValidator `json:"template_validator"`
```

Comment:
I would use `json:"templateValidator"` instead, I think it is easier on the eye and easier to type :)

Reply:
Changed. I see that other fields use camelCase too. The underscore version was added automatically by IDE.